data_quality.md

```diff
@@ -137,7 +137,7 @@ it executes on every column that's extracted.
 ## Handling the results
 
```

Comment:
this file needs to be merged into the main docs...

Reply:
(same with basics.md)

Reply:
Yep was waiting for the docs rewrite -- was planning to chop this commit then copy/paste.